circle.yml

```diff
@@ -17,6 +17,9 @@ dependencies:
     - echo y | android update sdk --no-ui --all --filter "tools"
     - echo y | android update sdk --no-ui --all --filter "build-tools-26.0.2"
     - yes | /usr/local/android-sdk-linux/tools/bin/sdkmanager --update --verbose
+    - wget https://dl.google.com/android/repository/android-ndk-r15c-linux-x86_64.zip; unzip android-ndk-r15c-linux-x86_64.zip:
```

Comment:
As per discussion below, ndk installation can be performed by sdk-manager?

https://discuss.circleci.com/t/licences-for-android-build-tools-not-accepted/17285/22

Reply:
Yes, but then I assume we'll get the latest NDK version, not a specific one?
Since NDK builds are often sensitive to the NDK version that is used, perhaps it's better to keep this as it is right now?

Reply:
With enough stars, perhaps CircleCI will provide this ready-made: 

https://github.com/circleci/circleci-images/issues/73